src/services/UnsService.ts

```diff
@@ -116,10 +117,14 @@ export class UnsService extends NamingService {
   }
 
   // key: 'address.${Uns key}'
-  async record (name: string, key: string): Promise<RecordItem> {
+  async record (name: string, key: string): Promise<RecordItem | null> {
     const recordItem = makeRecordItem(key)
     const unsKey = allDIDKeyToUnsKey(key)
-    recordItem.value = await this.uns.record(name, unsKey)
+    try {
+      recordItem.value = await this.uns.record(name, unsKey)
+    } catch {
+      return null
```

Comment:
We'd better check the error to determine if it is a EmptyError, and then return null.
Otherwise, we need to leave it to developers to decide what to do next.